axum/src/docs/routing/merge.md

```diff
@@ -1,4 +1,4 @@
-Merge two routers into one.
+Merge the paths of two routers into a single [`Router`].
```

Comment:
I was thinking about writing this as
```suggestion
Merge the paths and fallbacks of two routers into a single [`Router`].
```

but that doesn't make sense since there can only be one fallback